buck2/python/platforms/defs.bzl

```diff
@@ -45,6 +45,7 @@ def _platforms(ctx):
     return [
         DefaultInfo(),
         ExecutionPlatformRegistrationInfo(platforms = [platform]),
+        PlatformInfo(label = str(name), configuration = configuration),
```

Comment:
Same as before. Plus should this even be part of this PR?

Reply:
Without these lines the builds fail with the following error:
```
Expected `root//platforms:remote_platform` to be a `platform()` target, but it had no `PlatformInfo` provider.
```
Removed the `str)..) call`